src/components/Badges/TechnicalPreviewReleaseNote/TechnicalPreviewReleaseNote.test.tsx

```diff
@@ -0,0 +1,13 @@
+// THIS TEST IS BREAKING DUE TO JEST NOT FINDING THE @theme/ThemedImage
+
+// import React from "react";
```

Comment:
@nagesh161007, I'm curious if you encountered something similar to Jest when testing other components. Jest just can't seem to find `@theme/ThemedImage` 

```shell
npx jest src/components/Badges/TechnicalPreviewReleaseNote/TechnicalPreviewReleaseNote.test.tsx
 FAIL  src/components/Badges/TechnicalPreviewReleaseNote/TechnicalPreviewReleaseNote.test.tsx
  ● Test suite failed to run

    src/components/Badges/TechnicalPreviewReleaseNote/TechnicalPreviewReleaseNote.tsx:2:25 - error TS2307: Cannot find module '@theme/ThemedImage' or its corresponding type declarations.

    2 import ThemedImage from "@theme/ThemedImage";
```